kernel/arch/Makefile

```
@@ -5,15 +5,15 @@ include $(TOPDIR)/misc/mk/config.mk
 
 ifdef ARCH
 SUBDIRS:=$(ARCH)
+LO_DEPS=	$(ARCH)/$(ARCH).lo
 else
-SUBDIRS:=ppc sparc x86
+SUBDIRS:=arm ppc sparc x86
```

checkmake (0.2.2)

[warning] Target "SUBDIRS" should be declared PHONY.

(phonydeclared)